index.js

```diff
@@ -1,23 +1,63 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
 // Import of the model Recipe from './models/Recipe.model.js'
-const Recipe = require('./models/Recipe.model');
+const Recipe = require("./models/Recipe.model");
 // Import of the data from './data.json'
-const data = require('./data');
+const data = require("./data");
 
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+const MONGODB_URI = "mongodb://127.0.0.1:27017/recipe-app";
 
 // Connection to the database "recipe-app"
 mongoose
   .connect(MONGODB_URI)
-  .then(x => {
+  .then((x) => {
     console.log(`Connected to the database: "${x.connection.name}"`);
-    // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+    return Recipe.deleteMany();
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+  .then((newRecipe) => {
+    return Recipe.create({
+      title: "Paella",
+      level: "Amateur Chef",
+      ingredients: [
+        "1/2 cup rice vinegar",
+        "5 tablespoons honey",
+        "1/3 cup soy sauce (such as Silver Swan®)",
+        "1/4 cup Asian (toasted) sesame oil",
+        "3 tablespoons Asian chili garlic sauce",
+        "3 tablespoons minced garlic",
+        "salt to taste",
+        "8 skinless, boneless chicken thighs",
+      ],
+      cuisine: "Asian",
+      dishType: "main_course",
+      image:
+        "https://images.media-allrecipes.com/userphotos/720x405/815964.jpg",
+      duration: 40,
+      creator: "Chef LePapu",
+    });
   })
-  .catch(error => {
-    console.error('Error connecting to the database', error);
+  .then((newRecipe) => {
+    console.log(newRecipe.title);
+    return Recipe.insertMany(data);
+  })
+  .then((manyRecipes) => {
+    console.log(manyRecipes);
```

Comment:
You don't need a `.then()` just for the `console.log()` so you could have returned the next promise directly here
You could also use the `.forEach()` to print just the titles :) 
```suggestion
    manyRecipes.forEach(recipe => console.log(recipe.title))
      return Recipe.findOneAndUpdate(
      { title: "Rigatoni alla Genovese" },
      {duration:100},
      { new: true }
    );
```